e2e/test/scenarios/question/saved.cy.spec.js

```diff
@@ -268,6 +268,25 @@ describe("scenarios > question > saved", () => {
     cy.url().should("include", "/dashboard/");
   });
 
+  it("should not add scrollbar to duplicate modal if question name is long", () => {
+    H.createQuestion(
+      {
+        name: "A".repeat(240),
+        query: {
+          "source-table": ORDERS_ID,
+        },
+      },
+      { visitQuestion: true },
+    );
+    H.openQuestionActions();
+    H.popover().findByText("Duplicate").click();
+
+    H.modal().then($el => {
```

Comment:
To make it less flaky:

```suggestion
    H.modal().should($el => {
```